files/zh-tw/web/api/htmlinputelement/index.html

```diff
@@ -69,7 +69,7 @@ <h2 id="Properties" name="Properties">屬性</h2>
   <tr>
    <td><code>value</code></td>
    <td><code><em>string:</em></code> Returns / Sets the current value of the control.
-    <p class="note"><strong>Note:</strong> If the user enters a value different from the value expected, this may return an empty string.</p>
+    <p><strong>Note:</strong> If the user enters a value different from the value expected, this may return an empty string.</p>
```

Comment:
```suggestion
    <div class="notecard note">
     <p><strong>Note:</strong> If the user enters a value different from the value expected, this may return an empty string.</p>
    </div>
```